HelpAssistHer/help-assist-her
# Add id to url

## Description
Add id to url upon clicking "Verify next resource" button by using a `push` method on the `history` object in the click handler for `GetResourceToVerifyButton`. Added a new `Route` in `router.js` with `:id` wildcard.

## Test Plan
1. Go to PRC page.
2. Click "Verify Next Resource"
3. Open up the console and check to make sure that the id in the URL matches the resource in the state as well as the information displayed in the form.
(Added a `console.log('state',store.getState())` for testing purposes, will remove before merging.
